versions.properties

```diff
@@ -61,23 +59,17 @@ version.com.github.bumptech.glide..glide=4.13.1
 
 version.com.github.bumptech.glide..okhttp3-integration=4.13.1
 
-version.com.jakewharton.rxrelay2..rxrelay=2.0.0
-
-version.com.jakewharton.timber..timber=5.0.1
-
 version.com.nhaarman.mockitokotlin2..mockito-kotlin=2.2.0
 
 version.google.android.material=1.5.0
 
 version.google.dagger=2.41
 
-version.io.reactivex.rxjava2..rxandroid=2.0.2
-
-version.io.reactivex.rxjava2..rxjava=2.2.9
+version.jakewharton.rxrelay2=2.0.0
```

Comment:
`refreshVersions` tries to move dependencies around. this isn't changing anything, but just accepting the move so it's less noise for others in future